packages/table/src/table.tsx

```diff
@@ -489,10 +489,26 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
-    public resizeRowsByTallestCell(columnIndex: number) {
+    /**
+     * Resize all rows in the table to the height of the tallest visible cell in the specified columns.
+     * If no indices are provided, default to using the tallest cell from all columns currently in view.
```

Comment:
My mistake: do we need to clarify "tallest **visible** cell" in this sentence too, or is it implicitly clear?